### PR TITLE
PLATML-4570b dockerfile fix

### PR DESCRIPTION
**Pull request by shivmpatel12**
Dockerfile was no longer doing `python setup.py install` and `cp dist/retail*.egg /application.egg` meaning the egg was not being created

### CODE REVIEW

**Review by jonaphin — APPROVED** [review]
must be late because I fail to see when it disappeared... The history just stops showing the command, but does not show it as removed somehow.

**Comment by shivmpatel12** [comment]
Yeah I am also unclear on when it was changed. I think it was this PR but I'm not sure https://github.com/adobe/experience-platform-dsw-reference/pull/39

File: recipes/python/retail/Dockerfile [hunk 1]
```diff
@@ -17,14 +17,19 @@
 
 FROM adobe/acp-dsw-ml-runtime-python:0.26.2
```
Comment (rjrussell77) [inline recipes/python/retail/Dockerfile]: According to this: https://wiki.corp.adobe.com/display/AEPSRE/Dependent+Library+Management#DependentLibraryManagement-MLRuntimePythonCPU
the python runtime is at 0.27.0.  Can you verify there is a good reason why we should get latest here?

File: recipes/python/retail/Dockerfile [hunk 2]
```diff
@@ -17,14 +17,19 @@
...
 RUN /usr/bin/python3.5 -m pip install -U numpy==1.16.2
 RUN /usr/bin/python3.5 -m pip install -U pandas==0.24.2
 RUN /usr/bin/python3.5 -m pip install -U scikit-learn==0.20.3
 RUN /usr/bin/python3.5 -m pip install -U scipy==1.2.1
 RUN /usr/bin/python3.5 -m pip install -U fastparquet==0.2.1
 
-COPY dist/retail*.egg /application.egg
+WORKDIR /root/retail
+
+COPY . .
```
Comment (rjrussell77) [inline recipes/python/retail/Dockerfile:29]: Maybe for the future we copy only the py files needed to build the egg.

File: recipes/python/retail/Dockerfile (hunk 1 above)
Comment (rjrussell77) [inline recipes/python/retail/Dockerfile:18]: Update to 0.27.0 or merge other changes to get it.

**Review by rjrussell77 — APPROVED** [review]
LGTM

**Comment by rjrussell77** [comment]
Reminder to update our asset management wiki when completed.

**Pull request merged**